server/controllers/comment.js

```diff
@@ -0,0 +1,120 @@
+import models from "@models";
+import { httpResponse, serverError } from "@helpers/http";
+
+const { Comments } = models;
+
+/**
+ * @description Create a new Comment
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object}  Response message object
+ */
+export const createComment = async (req, res) => {
+  const articleId = req.article.id;
+  const { content } = req.body;
+  const { userId } = req.user;
+  try {
+    const newComment = await Comments.create({
+      articleId,
+      userId,
+      content
+    });
+
+    return httpResponse(res, {
+      statusCode: 201,
+      message: "Comment created successfully",
+      comment: newComment
+    });
+  } catch (error) {
+    return serverError(res, error);
+  }
+};
+
+/**
+ * @description Get all Comments
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object}  Response message object
+ */
+export const getAllComments = async (req, res) => {
+  const { comments } = req;
+
+  try {
+    return httpResponse(res, {
+      statusCode: 200,
+      success: true,
+      message: "Comments retrived successfully",
+      data: comments
+    });
+  } catch (error) {
+    return serverError(res, error);
+  }
+};
+
+/**
+ * @description Update a Comment
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object}  Response message object
+ */
+export const updateComment = async (req, res) => {
+  const { user } = req;
+  const { content } = req.body;
+  const { comment } = req;
```

Comment:
destructure this like this please  const {user, comment, body: {content} } = req